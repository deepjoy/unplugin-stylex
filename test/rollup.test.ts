import { expect, test } from 'vitest'
import stylexRollupPlugin from '../src/rollup'

test('Rollup should throw error', () => {
  expect(() => stylexRollupPlugin()).toThrowErrorMatchingSnapshot()
  expect(() => stylexRollupPlugin({ dev: true })).toThrowErrorMatchingSnapshot()
})
