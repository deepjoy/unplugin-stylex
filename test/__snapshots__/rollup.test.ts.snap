// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Rollup should throw error 1`] = `[Error: If you want to use this plugin through rollup, please use "@stylexjs/rollup-plugin" instead]`;

exports[`Rollup should throw error 2`] = `[Error: If you want to use this plugin through rollup, please use "@stylexjs/rollup-plugin" instead]`;
